site/docs/master/aws-config.md

```diff
@@ -196,113 +192,109 @@ For more complex installation needs, use either the Helm chart, or add `--dry-ru
-                "Action": [
-                    "ec2:DescribeVolumes",
-                    "ec2:DescribeSnapshots",
-                    "ec2:CreateTags",
-                    "ec2:CreateVolume",
-                    "ec2:CreateSnapshot",
-                    "ec2:DeleteSnapshot"
-                ],
-                "Resource": "*"
-            },
-            {
-                "Effect": "Allow",
-                "Action": [
-                    "s3:GetObject",
-                    "s3:DeleteObject",
-                    "s3:PutObject",
-                    "s3:AbortMultipartUpload",
-                    "s3:ListMultipartUploadParts"
-                ],
-                "Resource": [
-                    "arn:aws:s3:::${BUCKET}/*"
-                ]
-            },
-            {
-                "Effect": "Allow",
-                "Action": [
-                    "s3:ListBucket"
-                ],
-                "Resource": [
-                    "arn:aws:s3:::${BUCKET}"
-                ]
-            }
-        ]
-    }
-    EOF
-
-    aws iam put-role-policy \
-      --role-name velero \
-      --policy-name velero-policy \
-      --policy-document file://./velero-policy.json
-    ```
+```bash
+BUCKET=<YOUR_BUCKET>
+cat > velero-policy.json <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeVolumes",
+                "ec2:DescribeSnapshots",
+                "ec2:CreateTags",
+                "ec2:CreateVolume",
+                "ec2:CreateSnapshot",
+                "ec2:DeleteSnapshot"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetObject",
+                "s3:DeleteObject",
+                "s3:PutObject",
+                "s3:AbortMultipartUpload",
+                "s3:ListMultipartUploadParts"
+            ],
+            "Resource": [
+                "arn:aws:s3:::${BUCKET}/*"
+            ]
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListBucket"
+            ],
+            "Resource": [
+                "arn:aws:s3:::${BUCKET}"
+            ]
+        }
+    ]
+}
+EOF
+```
+```bash
+aws iam put-role-policy \
+  --role-name velero \
+  --policy-name velero-policy \
+  --policy-document file://./velero-policy.json
+```
 4. Update `AWS_ACCOUNT_ID` & `VELERO_ROLE_NAME` with `kubectl edit deploy/velero -n velero` and add the following annotation:
-
-    ```
-    ---
-    apiVersion: apps/v1beta1
-    kind: Deployment
-    metadata:
-        namespace: velero
-        name: velero
-    spec:
-        replicas: 1
-        template:
-            metadata:
-                labels:
-                    component: velero
-                annotations:
-                    iam.amazonaws.com/role: arn:aws:iam::<AWS_ACCOUNT_ID>:role/<VELERO_ROLE_NAME>
-    ...
```

Comment:
I think the point with the ellipsis here was that we aren't showing the entire deployment manifest, just the relevant subsection for this step.